heartbeat/beater/manager.go

```diff
@@ -189,15 +200,42 @@ func (m *Monitor) Update(configs []*common.Config) error {
 
 	// start new and reconfigured tasks
 	for id, t := range all {
-		job := createJob(m.manager.client, t)
-		t.cancel = m.manager.jobControl.Add(t.config.Schedule, id, job)
+		processors, err := processors.New(t.config.Processors)
+		if err != nil {
+			logp.Critical("Fail to load monitor processors: %v", err)
```

Comment:
This loop is about creating tasks, after merging the tasks settings (via json file) with the actual monitor settings. That is, this loop also drives monitors config reloading and is to be replaced is metricbeat/filebeat config reloading. In the filebeat/metricbeat case, the reloading is less flexible in handline settings, allowing us to load the configs once and not per task.